vignettes/changes.Rmd

```diff
@@ -20,26 +20,55 @@ these validations over the different versions.
 
 ## Specific changes
 
-### Since the original published validation (version 1)
+### Since the original validation (version from the paper)
 
 1.  We don't use purchases of semaglutid, dapagliflozin or
     empagliflozin, neither for inclusion events nor classification of
     diabetes type (due to increasing use in treatment of non-diabetes).
 2.  We no longer use diabetes type reclassification based on insulin
     purchases in the previous year.
+3.  The logic defining pregnancy index dates has been simplified to only
+    use diagnoses of pregnancy endings (no longer uses data on maternal
+    care visits).
+4.  De-duplicates subsequent HbA1c samples taken on the same date
+    (originally, if a sampling time was available in the lab data, only
+    samples taken at the same time were de-duplicated)
 
 ## Validity
 
 Algorithm validity across versions. Reports `PPV` (*positive predictive
 value*) and `sensitivity` for typical cases and cases with atypical age
 at onset of T1D (after age 40) and T2D (before age 40), respectively.
 
+### On pre-2019 data (as in the paper)
+
+**Overall and age at onset-stratified (paper table 1 & 2):**
+
+| Version | Diabetes type | PPV   | Sensitivity |
```

Comment:
Oh, do we want to do Semantic Versioning, so the first version (the paper version) is 1.0.0 and then this modified version is 1.0.1 (patch increase bc it's not adding functionality and it's only simplifying internal behaviour of the classifier)?

Reply:
I think the version refers to the algorithm, not the package. So semantic versioning might not exactly apply here.

Reply:
That's true! Nvm ☺️